src/engine/enginebuffer.cpp

```diff
@@ -1232,8 +1276,8 @@ void EngineBuffer::processSlip(int iBufferSize) {
         // TODO: Check if we can replace `iBufferSize` with the number of
         // frames per buffer in most engine method signatures to avoid this
         // back and forth calculations.
-        const int bufferFrameCount = iBufferSize / mixxx::kEngineChannelCount;
-        DEBUG_ASSERT(bufferFrameCount * mixxx::kEngineChannelCount == iBufferSize);
+        const int bufferFrameCount = iBufferSize / m_channelCount;
+        DEBUG_ASSERT(bufferFrameCount * m_channelCount == iBufferSize);
```

Comment:
Please adjust the comment above. It's no longer "a multiple of 2" but a "a multiple of channel count".